.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: ruby
 
 rvm:
-  - 2.2
+  - 2.3
```

Comment:
You need to use `2.3.1` for travis, I had the same problem with the [diaspora_federation-gem](https://github.com/diaspora/diaspora_federation/blob/develop/.travis.yml#L4)